package.json

```diff
@@ -63,10 +63,14 @@
 		"stylelint-config-recess-order": "5.0.1",
 		"stylelint-config-standard": "36.0.1",
 		"tsc-alias": "1.8.10",
-		"typescript": "5.5.4"
+		"typescript": "5.5.4",
+		"vite-plugin-svgr": "4.2.0"
```

Comment:
Not sure, if you installed `vite-plugin-svgr` into the correct place: do we need it for both frontend and backend?! and inot sure if it is a `dev dependency`: shouldn't it be in regular `dependencies` list?